google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -453,6 +468,21 @@ def autoclass_enabled= toggle
           patch_gapi! :autoclass
         end
 
+        ##
+        # Update method to update all attributes of autoclass of a bucket
+        # It accepts params as a Hash of attributes in the following format
+        # { enabled: 'true', terminal_storage_class: 'ARCHIVE' }
```

Comment:
Please either enclose this expression in backticks or indent it 4 spaces, so it gets rendered in code font. Otherwise yardoc will get confused by the curly braces and try to parse it as a cross-reference link.

Also, remove the single quotes around `'true'` (for enabled) since the value is supposed to be a boolean rather than a string. And change the quotes around `"ARCHIVE"` from single to double to match our preferred style.

Reply:
Done.